Client/Frontend/Browser/Tab Tray/Views/TabSyncCustomView.swift

```diff
@@ -132,6 +138,45 @@ class TabSyncHeaderView: UITableViewHeaderFooterView, TableViewReusable {
   }
 }
 
+extension TabSyncHeaderView: UIContextMenuInteractionDelegate {
+  public func contextMenuInteraction(_ interaction: UIContextMenuInteraction, configurationForMenuAtLocation location: CGPoint) -> UIContextMenuConfiguration? {
+    return UIContextMenuConfiguration(identifier: nil, previewProvider: nil) { [unowned self] _ in
+      var actionMenuChildren: [UIAction] = []
+
+      let allOpenAction = UIAction(
+        title: Strings.OpenTabs.openSessionOpenAllActionTitle,
+        image: UIImage(systemName: "plus"),
+        handler: UIAction.deferredActionHandler { _ in
+          self.delegate?.openAll(self, section: self.section)
+        })
+
+      let hideForAction = UIAction(
+        title: Strings.OpenTabs.openSessionHideAllActionTitle,
+        image: UIImage(systemName: "eye.slash"),
```

Comment:
Maybe use the brave one (`brave.eye.slash`) for consistency with Privacy Hub/Brave News menus